includes/derivatives.inc

```diff
@@ -216,38 +215,50 @@ function islandora_large_image_create_tn_derivative(AbstractObject $object, $for
     );
 
     $uploaded_file = islandora_large_image_get_uploaded_file($object);
-
-    $args = array();
-    $args[] = '-quality ' . escapeshellarg(variable_get('imagemagick_quality', 75));
-    $args[] = '-resize ' . escapeshellarg("200 x 200");
-    $args[] = '-colorspace RGB';
-    $derivative_file = islandora_large_image_imagemagick_convert($uploaded_file, "temporary://{$base_name}_TN.jpg", $args);
-    if ($derivative_file === FALSE) {
+    $colorspace = islandora_large_image_get_colorspace($uploaded_file);
+    if ($colorspace === FALSE) {
```

Comment:
Perhaps we should attempt to create the deriv file even if we can't find the colorspace.

Reply:
@DiegoPino [++](https://github.com/Islandora/islandora_solution_pack_large_image/pull/180#issuecomment-367032275)

Reply:
Hi, in this case its not the lack of colorspace. It is a warning (richtiff tag related) that because of the -regard-warnings i added returns 1. But i wonder if we should allow no colorspace images to run? Do they exist?

Reply:
@DiegoPino  go ahead I think we are on the same page.